test/documentation-content/bcd/4.0/antora.yml

```
@@ -0,0 +1,13 @@
+name: bcd
+title: Continuous Delivery
+# prerelease version but non alpha
+version: 4.0
```

Add quote, otherwise it is considered as a number and the `.0` part is removed
```suggestion
version: '4.0'
```